backend/sirius-components-charts/src/main/java/org/eclipse/sirius/components/charts/piechart/components/PieChartStyle.java

```diff
@@ -0,0 +1,171 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.charts.piechart.components;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import org.eclipse.sirius.components.annotations.Immutable;
+
+/**
+ * The style of a PieChart.
+ *
+ * @author fbarbin
+ */
+@Immutable
+public final class PieChartStyle {
+
+    private List<String> colors;
+
+    private Number strokeWidth;
```

Comment:
Why `strokeWidth` is a `Number` instead of `float` or `double`?
(see my other comment in `PieChartStyleProvider`)

Reply:
ok to be strict on the the type we will expect a float.